readme.md

```diff
@@ -262,6 +262,7 @@
 - [Storybook](https://github.com/lauthieb/awesome-storybook#readme) - Development environment for UI components.
 - [Blazor](https://github.com/AdrienTorris/awesome-blazor#readme) - .NET web framework using C#/Razor and HTML that runs in the browser with WebAssembly.
 - [Web Performance Metrics](https://github.com/csabapalfi/awesome-web-performance-metrics#readme) - Metrics to help understand page speed and user experience.
+- [Tailwind CSS](https://github.com/merchedhq/awesome-tailwindcss) - Utility-first CSS framework for rapid UI development.
```

Comment:
This should be https://github.com/aniftyco/awesome-tailwindcss

Reply:
Good catch, fixed it!